packages/lib-ethers/test/Liquity.test.ts

```diff
@@ -486,7 +448,7 @@ describe("EthersLiquity", () => {
         lusdLoss: Decimal.from(0),
         newLUSDDeposit: smallStabilityDeposit,
         collateralGain: Decimal.from(0),
-        lqtyReward: Decimal.from(0),
+        lqtyReward: undefined,
```

Comment:
this PR just removes the frontend tag, removing the frontend tag also removes some of the LQTY reward calcs/initialisation. Since the LQTY rewards will be removed it was easier to just match the test to output.